### ISSUE TITLE
No voice support in rockpoold

### ISSUE THREAD

**Issue by rufferson** [abranson/rockpool#39]
While attempting to use voice commands it logs 
[W] WatchConnection::readyRead:205 - Have message for unhandled endpoint 11000 "010000000001080001011d00312e327263310000000000000000000000000000803e00000032044001" 

that means voice endpoint implementation is missing 11000 = 0x2af8
 $ grep libpebble2/protocol/voice.py
    # Voice endpoint attribute (key-value pair)
        endpoint = 0x2af8
        endpoint = 0x2af8

**Comment by rufferson** [comment]
see [http://talk.maemo.org/showpost.php?p=1506043&postcount=387](url) for parsed hex string above

**Comment by strahlex** [comment]
Is this related to voice apps not working with the Pebble and Rockpool? It looks like the phone app has to do the actual voice recognition.

**Comment by rufferson** [comment]
yes, from what i see phone is responsible for recognition, watchapp is just receiving results of it and needs to interpret it whichever way it likes - either in jskit or directly in the watch.
